flex/interactive/docker/interactive-runtime.Dockerfile

```diff
@@ -21,26 +22,34 @@ RUN curl -sf -L https://static.rust-lang.org/rustup.sh | \
   cargo --version
 
 # install flex
-RUN . ${HOME}/.cargo/env  && cd ${HOME} && git clone https://github.com/alibaba/GraphScope.git -b main --single-branch && cd GraphScope/flex && \
+RUN . ${HOME}/.cargo/env  && cd ${HOME}/GraphScope/flex && \
```

Comment:
I suggest clone from main branch, rather than copy from current directory.

Reply:
By this way we can build images based on local changes.